### PR TITLE
Removed enrollment warnings, fixes DOC-1126

### PR DESCRIPTION
**Pull request by lamagnifica**
After discussion with @brianhw , most information about providing enrollment events for courses that started before 12/3/13 will go in release notes (separate repo and PR).
This PR removes the notes about enrollment for those courses, as all (+/-) courses will now display enrollment data.
@shnayder @mhoeber @WatsonEmily @catong @srpearce

### CODE REVIEW

**Comment by WatsonEmily** [comment]
:+1:

**Comment by srpearce** [comment]
:+1:

**Comment by catong** [comment]
:+1:

**Comment by lamagnifica** [comment]
@brianhw whenever you have a chance.

**Comment by lamagnifica** [comment]
@mulby , @clintonb , @shnayder waiting on thumbs from any one of you to release these doc updates.
@stroliova FYI

File: docs/en_us/dashboard/source/change_log.rst [hunk 1]
```diff
@@ -12,6 +12,9 @@ September, 2014
 
    * - Date
      - Change
+   * - 11/12/14
```
Comment (lamagnifica) [inline docs/en_us/dashboard/source/change_log.rst]: update

**Comment by clintonb** [comment]
:+1:

**Comment by stroilova** [comment]
thank you :+1:

**Pull request merged**